labrad/types.py

```diff
@@ -394,6 +389,28 @@ def _flatten_to(obj, types, endianness):
     return s, t
 
 
+def __flatten__(t, obj, endianness):
+    """Flatten an object to the specific given type.
```

Comment:
I am not a fan of this having a reversed call signature vs. flatten (obj, tt, endian) vs (t, obj, endian).  Also, we should avoid creating new __foo__ names, since those are reserved names in python.

Alternately, make this a method on the LRType base class, and then rename the existing __flatten__ methods to something like _flatten_real, which would avoid having to change all calls to t.__flatten__(...) to __flatten__(t, ...)